roles/kubernetes-apps/meta/main.yml

```diff
@@ -77,6 +77,13 @@ dependencies:
       - apps
       - persistent_volumes
 
+  - role: kubernetes-apps/container_runtimes
+    when:
+      - inventory_hostname == groups['kube-master'][0]
```

Comment:
role kubernetes-apps/container_runtimes always run ?
may be need add
when: kata_containers_enabled
?